src/pages/AdminBooksPage/BooksTable/index.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  Button,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TablePagination,
+  TableRow,
+  Typography,
+} from "@mui/material";
+import "./BooksTable.css";
+import { useState } from "react";
+import { EditOutlined } from "@mui/icons-material";
+import { useBooksContext } from "../../../store/books/catalog/BooksContext";
+import BookStatus from "../../../components/BookStatus";
+import BookStarRating from "../../../components/BookStarRating";
+
+const BooksTable = () => {
+  const { books, allBooksCount, page, setPage } = useBooksContext();
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  const handleChangePage = (event: unknown, newPage: number) => {
+    setPage?.(newPage);
+  };
+
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage?.(0);
+  };
+
+  return (
+    <div>
+      <TableContainer component={Paper}>
+        <Table aria-label="simple table" data-testid="books-table">
+          <TableHead>
+            <TableRow className="table-row">
+              <TableCell>
+                <Typography className="text">Image</Typography>
+              </TableCell>
+              <TableCell align="left">
+                <Typography className="text">Name</Typography>
+              </TableCell>
+              <TableCell align="left">
+                <Typography className="text">Author</Typography>
+              </TableCell>
+              <TableCell align="left">
+                <Typography className="text">Status</Typography>
+              </TableCell>
+              <TableCell align="left">
+                <Typography className="text">Rating</Typography>
+              </TableCell>
+              <TableCell />
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {books
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
```

Comment:
Currently, our endpoint for showing books is able to return paginated response with 18 books (18 is fixed number in our backend), so it should be fixed on frontend as well

Ideally we wouldn't do slice operation, try to think of way to be able to show paginated result in table without slicing the data
This is possible because each time we call search endpoint, 18 books are returned